antlr-kotlin-runtime/src/commonMain/kotlin/com/strumenta/antlrkotlin/runtime/SimpleBitSet.kt

```diff
@@ -139,10 +143,16 @@ internal class InternalBitSet(size: Int = ELEMENT_SIZE) {
     }
   }
 
+  /**
+   * Sets the bit at the specified [bitIndex] to `true`.
+   */
+  public fun set(bitIndex: Int): Unit =
+    set(bitIndex, true)
+
   /**
    * Sets the bit at the specified [bitIndex] to the specified [value].
    */
-  fun set(bitIndex: Int, value: Boolean = true) {
+  public fun set(bitIndex: Int, value: Boolean) {
```

Comment:
I think the default value made sense here, as setting a bit is a shorthand for setting the bit to true

Reply:
Also, other overloaded versions of the method use that convention

Reply:
The default value is no more there because we have a new method, just above, that accepts a single parameter and sets it to `true`.

This was done to make type-aliasing possible, otherwise the compiler was complaining about incompatible method signatures.

Reply:
Makes sense